DVTShowcaseColor.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DVTShowcaseColor.swift
+//  DVTShowcase
+//
+//  Created by Idris SOP on 2017/09/18.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+import UIKit
+
+extension UIColor {
+    convenience init(rgb: UInt) {
+        self.init(
+            red: CGFloat((rgb & 0xFF0000) >> 16) / 255.0,
+            green: CGFloat((rgb & 0x00FF00) >> 8) / 255.0,
+            blue: CGFloat(rgb & 0x0000FF) / 255.0,
+            alpha: CGFloat(1.0)
+        )
+    }
+}
+
+class DVTShowcaseColor: NSObject {
```

Comment:
Is there a specific reason why this class inherits from NSObject ? Inheriting a Swift class from NSObject causes it to be subject to a boatload of constraints from the Objective C object system. It's perfectly legitimate in Swift not to inherit from anything ( although there is a hidden Swift superclass ) unless one actually needs to be able to access the object from within Objective C.